### ISSUE TITLE
Make event metadata editable directly in Tobira

### ISSUE THREAD

**Issue by LukasKalbertodt** [elan-ev/tobira#371]
"My videos" has metadata controls disabled right now. To enable editing the metadata, we have to figure out the exact information flow here. But it's clearly something we want at some point.

This issue is just about the minimal "title" +  "description".

**Comment by oas777** [comment]
If this is for allowing users to edit metadata of their videos with the "Opencast editor", this is a requirement at ETH.

**Comment by LukasKalbertodt** [comment]
No, this is about editing it in Tobira directly. Editing through the Opencast editor should already work.

**Comment by oas777** [comment]
Where and what is "Tobira directly"? Something like https://tobira.opencast.org/~manage/videos/OS0wVcpHWjA?

**Comment by LukasKalbertodt** [comment]
Yes exactly. At some point, users should be able to edit metadata right there, without leaving Tobira. Right now you have to click "Open in video editor". And the video editor is a different UI and all that.

**Comment by LukasKalbertodt** [comment]
Implemented in #1374
